### PR TITLE
policy and terms page

### PR DESCRIPTION
**Pull request by dnidhi2710**

### CODE REVIEW

File: src/main/react-front-end/src/views/PolicyComponent.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    readTextFile = file => {
+        var rawFile = new XMLHttpRequest();
+        rawFile.open("GET", file, false);
+        rawFile.onreadystatechange = () => {
+            if (rawFile.readyState === 4) {
+                if (rawFile.status === 200 || rawFile.status == 0) {
+                    var allText = rawFile.responseText;
+                    console.log("allText: ", allText);
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/PolicyComponent.js]: This log can be removed or commented out

File: src/main/react-front-end/src/views/termsComponent.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    readTextFile = file => {
+        var rawFile = new XMLHttpRequest();
+        rawFile.open("GET", file, false);
+        rawFile.onreadystatechange = () => {
+            if (rawFile.readyState === 4) {
+                if (rawFile.status === 200 || rawFile.status == 0) {
+                    var allText = rawFile.responseText;
+                    console.log("allText: ", allText);
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/termsComponent.js]: This log can be removed or commented out

**Review by LinusCastelino — COMMENTED** [review]
Changes look fine. 
Just ensure that your editor is configured correctly. There are too many file updates with just spaces.
Need to discuss more on this tomorrow, so let's keep this PR on hold for now.

File: src/main/react-front-end/src/views/PolicyComponent.js (hunk 1 above)
Comment (dnidhi2710) [reply src/main/react-front-end/src/views/PolicyComponent.js]: done

File: src/main/react-front-end/src/views/termsComponent.js (hunk 2 above)
Comment (dnidhi2710) [reply src/main/react-front-end/src/views/termsComponent.js]: done

**Comment by LinusCastelino** [comment]
[UI page for ODS policies](https://trello.com/c/07szWowf/13-ui-page-for-ods-policies)

**Comment by aashish-jain** [comment]
Looks good!

**Review by LinusCastelino — APPROVED** [review]
Looks good.
Fine to merge

**Pull request merged**